### PR TITLE
Remove file server

### PR DESCRIPTION
**Pull request by hackdna**
Toward #2393 

Note: the rest of the code will be removed in the next release (to make sure that corresponding database tables are removed during migration in all instances like production, etc)

### CODE REVIEW

File: refinery/annotation_server/management/commands/add_genome_annotation.py [hunk 1]
```diff
@@ -16,7 +16,6 @@
 # Helper command to deal with additional annotation tracks not available from
 # UCSC genome browser
 #
-# TODO: Should we import annotation files into file_server?
 # how to cleanup directories used to store and download files
```
Comment (mccalluc) [inline refinery/annotation_server/management/commands/add_genome_annotation.py]: This line is probably a TODO in its own right?

File: refinery/visualization_manager/views.py [hunk 2]
```diff
@@ -535,16 +526,6 @@ def get_file_name(nodeuuid, samp_file=None, is_file_uuid=False):
                 FileStoreItem.MultipleObjectsReturned) as e:
             logger.error("Could not fetch FileStoreItem from %s: %s",
                          nodeuuid, e)
```
Comment (mccalluc) [inline refinery/visualization_manager/views.py]: With this change, have our expectations around `is_file_uuid` changed? As it was, either temp_fs would have been set, or there would have been an error, and that's no longer the case.

File: refinery/annotation_server/management/commands/add_genome_annotation.py (hunk 1 above)
Comment (hackdna) [reply refinery/annotation_server/management/commands/add_genome_annotation.py]: I was confused about this line. Should I prefix it with a TODO?

File: refinery/visualization_manager/views.py (hunk 2 above)
Comment (hackdna) [reply refinery/visualization_manager/views.py]: This whole function (`create_igv_session`) appears to be used only by `igv_session` view which in turn appears to be unused (it looks like it was used for Java-based IGV).

Comment (scottx611x) [reply refinery/visualization_manager/views.py]: We definitely don't need `create_igv_session` anymore I'd remove it and any usages as well

**Comment by hackdna** [comment]
@scottx611x We'll need to remove file_server entries from the django_migrations table (`manage.py migrate --fake file_server zero`) from production and other instances after this release is deployed. Then the next release would contain the removal of the rest of the file_server code.

I've decided to keep `create_igv_session` to limit the scope of this PR - it will be removed as part of #2464.

**Pull request merged**